src/main.ts

```diff
@@ -153,13 +193,26 @@ const logAttestation = (
 }
 
 // Attach summary information to the GitHub Actions run
-const logSummary = async (attestation: AttestResult): Promise<void> => {
-  const { attestationID } = attestation
-
-  if (attestationID) {
-    const url = attestationURL(attestationID)
-    core.summary.addHeading('Attestation Created', 3)
-    core.summary.addList([`<a href="${url}">${url}</a>`])
+const logSummary = async (attestations: AttestResult[]): Promise<void> => {
+  if (attestations.length > 0) {
+    core.summary.addHeading(
+      /* istanbul ignore next */
+      attestations.length !== 1
+        ? 'Attestations Created'
+        : 'Attestation Created',
+      3
+    )
+    const listItems: string[] = []
+    for (const attestation of attestations) {
+      if (attestation.attestationID) {
+        const url = attestationURL(attestation.attestationID)
+        for (const subject of attestation.attestationSubjects) {
+          const digest = formatSubjectDigest(subject)
+          listItems.push(`<a href="${url}">${subject.name}@${digest}</a>`)
+        }
```

Comment:
This change means that each subject in the attestation is always listed in the step summary, regardless of the single-subject option value.

Reply:
Why print one link per subject name? For the default use case this might be a bit confusing since it'll generate multiple links that all point to the same url (since the urls are per attestationID).

Let's keep this as it was.

As a benefit, as far as I can tell this is the only place we use attestationSubjects?, so if we back out this change (aside from attestations.length) I think we can get rid of the change you made to the return type in `attest.ts`

Reply:
More to expose the subject names for a sanity check when using a subject path with a glob (see e.g. https://github.com/sphinx-doc/sphinx/actions/runs/13466973845). But not essential, so reverted to only print one link per attestation.

> this is the only place we use attestationSubjects

Not quite, also used in `logAttestation()` (see other comment)